tests/Behat/Content/Node/ProjectContentContext.php

```diff
@@ -0,0 +1,78 @@
+<?php
```

Comment:
can you test if we need this file ? try removing and run the behat tests, if they are green, then commit the removal

Reply:
Yeah it works. Changed doing `git commit --amend`

I don't really understand the existence of tests/Behat/Content/Node/**Event**ContentContext.php